packages/server/database/types/Reflection.ts

```diff
@@ -55,6 +58,7 @@ export default class Reflection {
     this.content = content
     this.plaintextContent = plaintextContent || extractTextFromDraftString(content)
     this.entities = entities
+    this.sentimentScore = sentimentScore || 0.0
```

Comment:
-1 I think we should distinguish neutral meetings from having no data, i.e. leave it undefined if we don't have a score.